components/sensor/bh1750.rst

```diff
@@ -59,6 +59,5 @@ See Also
 
 - :ref:`sensor-filters`
 - :doc:`tsl2561`
```

Comment:
Looks like it was just misspelt and should be this which links [here](https://esphome.io/api/bh1750_8h.html)
```suggestion
- :doc:`tsl2561`
- :apiref:`bh1750/bh1750.h`
```